### PR TITLE
fix: show only accepted and confirmed session

### LINKED ISSUE #3752: My-session: session is showing multiple times
**Describe the bug**
My-session: session is showing multiple times 

**Expected behaviour**
session should show only once

**Screenshots**
[multiple session](http://www.youtube.com/watch?v=EMXLyCehYCM "multiple session")

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**
i want to work on this

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes #3752 
Screenshot from 2020-01-03 17-05-05

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/templates/my-sessions/list.hbs [hunk 1]
```diff
@@ -1,15 +1,12 @@
 <div class="row">
   <div class="sixteen column wide">
     {{#each model as |session|}}
-      {{session-card session=session}}
+      {{#if (or (eq session.status 'confirmed')(eq session.status 'accepted'))}}
```
Comment (kushthedude) [inline app/templates/my-sessions/list.hbs]: This is not the right approach, It is quite inefficient as well. Only accepted and confirmed sessions should be queried and displayed

Comment (maze-runnar) [reply app/templates/my-sessions/list.hbs]: So this would be solve by serve or by frontend ?

Comment (maze-runnar) [reply app/templates/my-sessions/list.hbs]: So would i have to do changes in model or something like that for quering only confirmed and accepted sessions?

Comment (uds5501) [reply app/templates/my-sessions/list.hbs]: @maze-runnar it should be handled on server level

**Comment by kushthedude** [comment]
Correctly stated by @uds5501 .
Open an issue on server for this.

**Comment by kushthedude** [comment]
https://github.com/fossasia/open-event-frontend/issues/3752#issuecomment-570923055

**Comment by maze-runnar** [comment]
@kushthedude original issue was that one session was showing multiple time in the list , i fixed that .

**Comment by kushthedude** [comment]
No we dont have to filter, my session is only the place where speaker can
get details about his submitted sessions, updates about them. If we will
filter them it will create a huge chaos.

**Pull request closed without merging**